network-api/networkapi/templates/donate/pages/base.html

```diff
@@ -30,12 +30,16 @@
 
 {% block bodyclass %} donate tw-m-0 tw-font-sans tw-font-normal tw-text-base tw-font-normal tw-text-black tw-bg-white tw-scroll-smooth {% endblock %}
 
-{% block donate_banner %}{% endblock %}
+{% block donate_banner %}
+    {% if settings.donate_banner.SiteDonateBanner.active_donate_banner %}
+        {% include "partials/donate_banner.html" with banner=settings.donate_banner.SiteDonateBanner.active_donate_banner.localized %}
+    {% endif %}
+{% endblock %}
 
 {% block sticky_top_class %} tw-sticky tw-z-[1020] tw-top-0 {% endblock %}
 
 {% block primary_nav %}
-    {% include "../fragments/nav.html" %}
+    {% include "partials/primary_nav.html" %}
```

Comment:
Per request of @kristinashu, updating the nav to match the rest of the foundation site! [link to slack thread](https://mozilla.slack.com/archives/C04DYP6T6VD/p1699560239000609)